Tests/SentryTests/SentrySDKTests.swift

```diff
@@ -194,6 +194,17 @@ class SentrySDKTests: XCTestCase {
         XCTAssertIdentical(scope, SentrySDK.currentHub().scope)
     }
     
+    func testDontStartInsideXcodePreview() {
+        SentrySDK.processInfoEnvironment = ["XCODE_RUNNING_FOR_PREVIEWS": "1"]
+        
+        SentrySDK.start { options in
+            options.debug = true
+            options.removeAllIntegrations()
```

Comment:
Is there some significance to calling this method?

Reply:
Not really no. Ctrl+c Ctrl+v from another test.

Reply:
```suggestion
```